test_files/preprocessor.h

```diff
@@ -0,0 +1,4 @@
+
+#define FOO 0
+#define FOOBAR 1
+#define FOOBARBAZ 2
```

Comment:
What happens if the define is not an integer? Do we need to check the actual token kind/type of the second token to make sure it can be translated to D?